src/views/migration/constant.ts

```diff
@@ -0,0 +1,36 @@
+import { K8sModel } from '@openshift-console/dynamic-plugin-sdk';
+import { K8sResourceCommon } from '@openshift-console/dynamic-plugin-sdk';
+
+export const MTV_OPERATOR = 'mtv-operator';
+
+export type PackageManifestKind = K8sResourceCommon & { status: any };
+
+export type SubscriptionsKind = K8sResourceCommon & { spec: any; status: any };
+
+export const SubscriptionModel: K8sModel = {
```

Comment:
we should not add models in the plugin code, they get stale and we can't update them manually each time upstream update a version.

Reply:
Should I use any or maybe we can create these models in the kubevirt-api in some ways?  Same thing for the SubscriptionsKind

Reply:
kubevirt-api, in the short term we can add it as hard coded modals, in the longer term we will need some automation to auto update the models